freeipa/src/main/resources/freeipa-salt/salt/freeipa/common-install.sls

```diff
@@ -37,6 +37,18 @@ add-httpd-x-cdp-trace-id:
     - source: salt://freeipa/scripts/getkeytab.py
     - onlyif: test -f /etc/ipa/default.conf
 
+/usr/lib/python2.7/site-packages/ipaserver/plugins/stageuser.py:
+  file.patch:
+    - source: salt://freeipa/scripts/stageuser.patch
+    - hash: md5=8a488621d3d8c16677133f1f32810325
+    - onlyif: test -f /etc/ipa/default.conf
+
+/usr/lib/python2.7/site-packages/ipaserver/plugins/user.py:
+  file.patch:
+    - source: salt://freeipa/scripts/user.py.patch
+    - hash: md5=0fdd233451b47c481464c0fda24b7be3
+    - onlyif: test -f /etc/ipa/default.conf
```

Comment:
these changes should be applied during image burning, so it should be part of `cloudbreak-images` repository
I see there are some states before this with similar functionality, I don't know why are they here

Reply:
yes, I followed the pattern for installation of the `getkeytab` plugin. This PR is just the code that I used for the redhat engagement. I wanted to throw it up here in a PR to get this into the codebase faster.

Can I just move these individual files over  to `cloudbreak-images` or are y'all expecting this to be packaged into an rpm? I'm not really familiar with that repo nor have I built many rpms.

Reply:
thats fine having these patches there, no need for rpms. we use saltstack to do the initial configuration, so this is almost a cut and paste operation to move these over